#request fields
string req
---
#response fields
bool autodetected

